src/ui/configuration/nodeDebugConfigurationResolver.ts

```diff
@@ -45,6 +47,8 @@ export class NodeConfigurationResolver extends BaseConfigurationResolver<AnyNode
   constructor(
     @inject(ExtensionContext) context: vscode.ExtensionContext,
     @inject(INvmResolver) private readonly nvmResolver: INvmResolver,
+    // We don't need Node to support remote file system at the moment, so we hardcode fsUtils
```

Comment:
What I would do instead for these cases is inject the local FS utils in `createGlobalContainer`, since we know that for services that operate independently of a DAP connection we will never be able to do anything else. Then in these cases you can inject the fs utils normally.

Reply:
Done